tests/helpers/test_debounce.py

```diff
@@ -20,17 +20,22 @@ async def test_immediate_works(hass):
     assert len(calls) == 1
     assert debouncer._timer_task is not None
     assert debouncer._execute_at_end_of_timer is False
+    assert debouncer._job.target == debouncer.function
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6309587)